backstop/tests/text-field.html

```diff
@@ -302,37 +312,37 @@ <h2>Hybrid layouts</h2>
 
     <h2>Without label</h2>
     <section id="demo-no-label">
-      <label class="iui-input-container">
+      <div class="iui-input-container">
```

Comment:
Why changed to `div`?

Reply:
![image](https://user-images.githubusercontent.com/9084735/140379476-d5abf86c-8c04-471d-b94f-3a2422b3d1a7.png)

Doesn't make sense to use `label` when there's no label